src/core/MainWindow.cpp

```diff
@@ -1576,6 +1576,13 @@ void MainWindow::on_actionIssue_triggered()
     openIssue();
 }
 
+void MainWindow::on_actionDocumentation_triggered()
+{
+    QString url;
+    url = "https://cutter.re/docs/user-docs.html";
+    QDesktopServices::openUrl(QUrl(url, QUrl::TolerantMode));
```

Comment:
If you are making an URL, make an URL no need to do it in 3 steps.
```suggestion
    QUrl url("https://cutter.re/docs/user-docs.html");
    QDesktopServices::openUrl(url);
```
or even
```suggestion
    QDesktopServices::openUrl(QUrl("https://cutter.re/docs/user-docs.html"));
```

Either way there is no need to explicitly specify the URL parsing mode if you don't have anything weird in it. Default mode will do just fine.